util/util.go

```diff
@@ -26,9 +26,10 @@ const (
 	ControllerServiceName = "controller"
 	ReplicaServiceName    = "replica"
 
-	AWSAccessKey = "AWS_ACCESS_KEY_ID"
-	AWSSecretKey = "AWS_SECRET_ACCESS_KEY"
+	AWSAccessKey      = "AWS_ACCESS_KEY_ID"
+	AWSSecretKey      = "AWS_SECRET_ACCESS_KEY"
 	BackupStoreTypeS3 = "s3"
+	AWSEndPoint       = "AWS_ENDPOINTS"
```

Comment:
This looks like a weird sandwich... Can you put `BackupStoreTypeS3` above the `AWSAccessKey`?